src/components/main/actionsPanel/workspaceTreeView/hooks/useNodeActionsHandler.ts

```diff
@@ -202,7 +217,16 @@ export const useNodeActionsHandler = ({
         // add to event history
         const _fileAction: TFileAction = {
           action: "move",
-          payload: { uids },
+          payload: {
+            uids: uids.map((uid, index) => ({
+              orgUid: uid,
+              newUid: `${targetNode.parentUid}/${
```

Comment:
we shouldn't use the `/` character directly.
instead, we can use `_path.join` or `_path.sep`
you can check my code by searching. thanks.